src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/BurdenEstimates/RetrieveBurdenEstimateTests.kt

```diff
@@ -66,6 +69,311 @@ class RetrieveBurdenEstimateTests : BurdenEstimateTests()
+        Assertions.assertThat(firstRow[6]).isEqualTo("cases")
+        Assertions.assertThat(firstRow[7]).isEqualTo("dalys")
+        Assertions.assertThat(firstRow[8]).isEqualTo("deaths")
+
+
+        var dataRow = csv.drop(1).first().toList()
+        Assertions.assertThat(dataRow.count()).isEqualTo(9)
+        Assertions.assertThat(dataRow[0]).isEqualTo("Hib3")
+        Assertions.assertThat(dataRow[1]).isEqualTo("2000")
+        Assertions.assertThat(dataRow[2]).isEqualTo("20")
+        Assertions.assertThat(dataRow[3]).isEqualTo("ABC")
+        Assertions.assertThat(dataRow[4]).isEqualTo("ABC-Name")
+        Assertions.assertThat(dataRow[5]).isEqualTo("100.0")
+        Assertions.assertThat(dataRow[6]).isEqualTo("40.0")
+        Assertions.assertThat(dataRow[7]).isEqualTo("25.5")
+        Assertions.assertThat(dataRow[8]).isEqualTo("2.0")
+
+        dataRow = csv.drop(2).first().toList()
+        Assertions.assertThat(dataRow.count()).isEqualTo(9)
+        Assertions.assertThat(dataRow[0]).isEqualTo("Hib3")
+        Assertions.assertThat(dataRow[1]).isEqualTo("2000")
+        Assertions.assertThat(dataRow[2]).isEqualTo("20")
+        Assertions.assertThat(dataRow[3]).isEqualTo("DEF")
+        Assertions.assertThat(dataRow[4]).isEqualTo("DEF-Name")
+        Assertions.assertThat(dataRow[5]).isEqualTo("50.0")
+        Assertions.assertThat(dataRow[6]).isEqualTo("10.0")
+        Assertions.assertThat(dataRow[7]).isEqualTo("5.5")
+        Assertions.assertThat(dataRow[8]).isEqualTo("1.0")
+
+        dataRow = csv.drop(3).first().toList()
+        Assertions.assertThat(dataRow.count()).isEqualTo(9)
+        Assertions.assertThat(dataRow[0]).isEqualTo("Hib3")
+        Assertions.assertThat(dataRow[1]).isEqualTo("2000")
+        Assertions.assertThat(dataRow[2]).isEqualTo("21")
+        Assertions.assertThat(dataRow[3]).isEqualTo("ABC")
+        Assertions.assertThat(dataRow[4]).isEqualTo("ABC-Name")
+        Assertions.assertThat(dataRow[5]).isEqualTo("200.0")
+        Assertions.assertThat(dataRow[6]).isEqualTo("80.0")
+        Assertions.assertThat(dataRow[7]).isEqualTo("35.5")
+        Assertions.assertThat(dataRow[8]).isEqualTo("3.0")
+
+        dataRow = csv.drop(4).first().toList()
+        Assertions.assertThat(dataRow.count()).isEqualTo(9)
+        Assertions.assertThat(dataRow[0]).isEqualTo("Hib3")
+        Assertions.assertThat(dataRow[1]).isEqualTo("2001")
+        Assertions.assertThat(dataRow[2]).isEqualTo("20")
+        Assertions.assertThat(dataRow[3]).isEqualTo("ABC")
+        Assertions.assertThat(dataRow[4]).isEqualTo("ABC-Name")
+        Assertions.assertThat(dataRow[5]).isEqualTo("150.0")
+        Assertions.assertThat(dataRow[6]).isEqualTo("60.0")
+        Assertions.assertThat(dataRow[7]).isEqualTo("30.5")
+        Assertions.assertThat(dataRow[8]).isEqualTo("4.0")
+
+    }
+
+    @Test
+    fun `can get burden estimate data with some missing outcome values`()
+    {
```

Comment:
Blackbox tests take longer to run cos they're making requests to the API, so it'd generally be better to test logic like this in the Database tests.

Reply:
Fair enough, though what these tests are really testing is the logic coordinated by the BurdenEstimateLogic class in putting the actual BurdenEstimateOutcomes together with the expectations - so I'll move these tests into the unit tests for the logic class.

Reply:
Even better, the logic test are the quickest and easiest to run :)